runtime-builder/tests/integration/test.sh

```diff
@@ -34,7 +34,8 @@ if [[ "${use_rb}" = "False" || "${rb_root}" != file://* ]]; then
     exit 1
 fi
 
-: ${STAGING_BUILDER_IMAGE?"Staging builder image path not set."}
+: ${TAG?"Staging builder image tag not set."}
```

Comment:
I'd like to have TAG to be set by passing in an argument instead of using environment variable.  Also, nice to have it as optional argument, i.e. default to staging if not passed in.

Please update usage to indicate this argument too.

Thanks.